DiffPriv/__init__.py

```diff
@@ -1,3 +1,10 @@
+"""
+DiffPriv is a package focused on privacy, with differential privacy and encryption schemes.
+
+.. include:: ../README.md
+"""
+__docformat__ = "restructuredtext"

```

Comment:
Is this compatible with pdoc? As far as I know, pdoc*3* only supports Markdown with rst *directives*

Reply:
This isn't pdoc*3* this is pdoc. And *pdoc* does support it.